### DISCUSSION TITLE
Keys

### DISCUSSION THREAD

**Discussion by Prachethana** [skyline-emu/skyline#153] [Q&A] [answered]
Hi how can to get all the keys for emulator

**Comment by Sssnowyyy** [comment, marked as answer]
https://yuzu-emu.org/help/quickstart/#dumping-prod-keys-and-title-keys Follow this

**Comment by IProXD** [comment]
How do i get key

**Reply by Gamers0996** [reply]
For skyline this person means

**Reply by chostdz** [reply]
https://www.mediafire.com/file/059jdmxxlh0jhpr/Prod_and_Title_Keys_14.1.2.rar/file
